src/app/legacy/containers/SocialEmbed/index.jsx

```diff
@@ -13,22 +13,34 @@ import { GridItemMedium } from '#components/Grid';
 import { socialEmbedBlockPropTypes } from '#models/propTypes/socialEmbed';
 import nodeLogger from '#lib/logger.node';
 import { SOCIAL_EMBED_RENDERED } from '#lib/logger.const';
+import isLive from '#lib/utilities/isLive';
 import createTranslations from './common/translations';
 import { LAZYLOAD_OFFSET, Wrapper } from './common/styles';
 import { getProviderFromSource, getIdFromSource } from './sourceHelpers';
 
 const logger = nodeLogger(__filename);
 
-const SocialEmbedContainer = ({ blocks, source }) => {
+const SocialEmbedContainer = ({ blocks, source, pageType }) => {
   const { isAmp } = useContext(RequestContext);
```

Comment:
`pageType` is available here - doesn't need to be a new prop:
```suggestion
  const { isAmp, pageType } = useContext(RequestContext);
```
Can be compared to the string in `import { ARTICLE_PAGE } from '#app/routes/utils/pageTypes';`